pkgs/games/minosoft/default.nix

```
@@ -0,0 +1,101 @@
+{ lib, stdenv
+, makeWrapper
+, fetchFromGitLab
+, gradle
+, perl
+, jre
+, libpulseaudio
+, makeDesktopItem
+, copyDesktopItems
+}:
+
+let
+  pname = "minosoft";
+  version = "7ba4569"; # there are no tags yet, so lets version this using tags for now
```

In such case, you must use 
```suggestion
  version = "unstable-YYYY-MM-DD";
```

where the `YYYY-MM-DD` is the date of the commit you are picking.

---

will do